### ISSUE TITLE
German translation of the "verify your password"-mail is broken

### ISSUE THREAD

**Issue by susnux** [nextcloud/appstore#931]
## Details
After signing in using github an email is sent, but the German translation is broken:

```
Benötigt keine Übersetzung. Für den Appstore wird nur die formelle Übersetzung verwendet (de_DE).Nextcloud App Storesusnuxapps.nextcloud.comhttps://apps.nextcloud.com/confirm-email/----/apps.nextcloud.com

Vielen Dank von Nextcloud App Store!

apps.nextcloud.com
```

Instead of some meaningful text, a message is shown "no translation needed".

## Steps to reproduce

1. Open appstore
2. login first time using github
3. receive weird verify your password mail

**Comment by juliushaertl** [comment]
Right, seems this was fixed already 9941d5460cf2dfc9d43f9053610e6eca4efafc6a Sorry for the noise

**Comment by susnux** [comment]
@juliushaertl this commit is from Oct. 2021, but I faced this issue few days before opening, so Oct. 2022.

So this might be a deployment problem?
(But I have no idea how and when the AppStore is updated, as the latest tag is from 2018).

**Comment by juliushaertl** [comment]
Sorry then I missed looking at the year 🙈 Maybe @mgallien can tell which commit is currently deployed.

**Comment by susnux** [comment]
> Last september there was another related issue and @CarlSchwan was working on it.
>
> :framed_picture: 

I noticed this translation issue today too, probably really some deployment issue.

**Comment by Nuube** [comment]
Is there any update or any workaround? 

Basically registration is currently not possible .

**Comment by mgallien** [comment]
@juliushaertl 
the currently deployed commit is afe6243099b74e47fd2ef304f1eba57036ce3d42

**Comment by juliushaertl** [comment]
Deployment is from 29th November 2022 but that commit doesn't contain this string anymore.

@mgallien Can you check if https://nextcloudappstore.readthedocs.io/en/latest/translation.html#deploying-translations is part of the upgrade steps for the App Store? Apart from that I think running `python manage.py compilemessages` should bring those up to date then.

**Comment by Nuube** [comment]
The problem is not only that the translation is missing / does not make any sense, but the registration link is not working.

At least not with my email reader.

**Comment by Nuube** [comment]
This is related. What I get via email is:

```
Benötigt keine Übersetzung. Für den Appstore wird nur die formelle Übersetzung verwendet (de_DE).Nextcloud App StoreMichael@xxxxxx.xxx.netapps.nextcloud.comhttps://apps.nextcloud.com/confirm-email/xxxxxxxxxxxxxxxk6ikgofsgjz4v3pljix8r4pvxg0xxxxzmrbwkvtmdrvbkrqag/apps.nextcloud.com

Vielen Dank von Nextcloud App Store!

apps.nextcloud.com
```

There are spaces missing before the link so you can not click on the link to open it.

I tried retrieving the link starting from "https" by copy&paste but was not succesfull

**Comment by Nuube** [comment]
I had to guess what is part of the link and what not.


From the string

``StoreMichael@xxxxxx.xxx.netapps.nextcloud.comhttps://apps.nextcloud.com/confirm-email/xxxxxxxxxxxxxxxk6ikgofsgjz4v3pljix8r4pvxg0xxxxzmrbwkvtmdrvbkrqag/apps.nextcloud.com``

so I guessed

``https://apps.nextcloud.com/confirm-email/xxxxxxxxxxxxxxxk6ikgofsgjz4v3pljix8r4pvxg0xxxxzmrbwkvtmdrvbkrqag/apps.nextcloud.com``

but possibly it could have been

``StoreMichael@xxxxxx.xxx.netapps.nextcloud.comhttps://apps.nextcloud.com/confirm-email/xxxxxxxxxxxxxxxk6ikgofsgjz4v3pljix8r4pvxg0xxxxzmrbwkvtmdrvbkrqag``

as well (I did not try). But what is this prefix

``StoreMichael@xxxxxx.xxx.netapps.nextcloud.com``

then doing here?

**Comment by bigcat88** [comment]
> Last september there was another related issue and @CarlSchwan was working on it.
> 
> <img alt="Bildschirmfoto 2022-09-04 um 17 58 09" width="1197" src="https://user-images.githubusercontent.com/2069590/211557850-088ed422-7dbd-473d-8056-61aea13fecb0.png">
> 
> Unfortunately I cannot find the issue right now. Maybe it was at cloud.nextcloud.com
> 
> To me this also looks like an deployment thing. Missing to convert PO to MO files was also one thought.

<img width="610" alt="image" src="https://github.com/nextcloud/appstore/assets/13381981/d1c019b8-b418-4e7c-83b7-ae6af856db4b">


Any idea how I can reproduce the issue?
